webapp/src/ts/providers/analytics-route-guard.provider.ts

```diff
@@ -0,0 +1,31 @@
+import { ActivatedRouteSnapshot, Router, CanActivateChild, UrlTree } from '@angular/router';
```

Comment:
I think it'd be nice to name this files like: `analytics.guard.provider.ts`
There's another guard named `route-guard.provider.ts` so it can be: `app.guard.provider.ts`

What do you think?

Reply:
@dianabarsan what do you think about this naming? Is it worth it?

Reply:
What if we'd put all route guard providers in one single file?

Reply:
Yeah, it could get long
Adding `route` in the name would be something like: `analytics-route.guard.provider.ts` 
The `app-route.guard.provider.ts` should probably be in the same level as app.component.ts

Reply:
I've done the rename of these files